controllers/memberstatus/memberstatus_controller.go

```diff
@@ -295,25 +295,27 @@ func (r *Reconciler) cheHandleStatus(reqLogger logr.Logger, memberStatus *toolch
 	}
 
 	if !r.isCheAdminUserConfigured(config) {
-		err := fmt.Errorf("Che admin user credentials are not configured but Che user deletion is enabled")
+		err := fmt.Errorf("the Che admin user credentials are not configured but Che user deletion is enabled")
 		errCondition := status.NewComponentErrorCondition(toolchainv1alpha1.ToolchainStatusMemberStatusCheAdminUserNotConfiguredReason, err.Error())
 		memberStatus.Status.Che.Conditions = []toolchainv1alpha1.Condition{*errCondition}
 		return err
 	}
 
-	// Get che route for testing user API
+	// Ensure it's possible to construct the che URL for using the Che user API
 	if _, err := r.cheDashboardURL(config); err != nil {
 		wrappedErr := errs.Wrapf(err, "Che dashboard URL unavailable but Che user deletion is enabled")
 		errCondition := status.NewComponentErrorCondition(toolchainv1alpha1.ToolchainStatusMemberStatusCheRouteUnavailableReason, wrappedErr.Error())
 		memberStatus.Status.Che.Conditions = []toolchainv1alpha1.Condition{*errCondition}
 		return err
 	}
 
-	// User API check
-	if err := r.CheClient.UserAPICheck(); err != nil {
-		errCondition := status.NewComponentErrorCondition(toolchainv1alpha1.ToolchainStatusMemberStatusCheUserAPICheckFailedReason, err.Error())
-		memberStatus.Status.Che.Conditions = []toolchainv1alpha1.Condition{*errCondition}
-		return err
+	// User API check (not applicable after migration from Che to Dev Spaces)
+	if config.Che().Namespace() != "crw" && config.Che().RouteName() != "devspaces" {
```

Comment:
could we have something like `config.Che().IsDevSpaces()` to simplify the code/abstract the logic here?

Reply:
That's a good suggestion thanks! Done in d9f8ea8d5cc43844df8c6f2c5f2f27c1e4a0dc69 and 2a51ae5758897b4faabb53c0fe843bcad6e1b677